template/template.go

```diff
@@ -37,6 +37,11 @@ type ResolvedValue struct {
 	err       error
 }
 
+// Value returns raw value.
+func (r *ResolvedValue) Value() any {
```

Comment:
unused?

Reply:
It's for external devs to get to the raw value.

Reply:
internally, we access ResolvedValue.value

Reply:
Does the current implementation require it? This PR. :)